docs/OV_Runtime_UG/automatic_batching.md

```diff
@@ -0,0 +1,40 @@
+
+Notice that _automatic batch size selection_ assumes that the application queries the ov::optimal_number_of_infer_requests to create and runs simultaneously the returned number of requests. This is very important, as there is a "timeout" value so that (when no enough inputs have been collected) the execution transparently falls back to execution the requests individually. Configuration-wise this is AUTO_BATCH_TIMEOUT property, please see below.
+The timeout, which adds itself to the execution time of the requests, heavily penalizes the performance. To avoid it, in the cases when your parallel slack is bounded (e.g. the application processes only 4 video-streams, so batch larger than 4 doesn't make sense) make sure to give the OpenVINO an additional hint the OV on this.
+When the performance hint is equipped with the optional ov::hint::num_requests configuration key set to "4". This is the most future-proof way to communicate the limitations on the parallelism. E.g. for the GPU that will limit the batch size, for the CPU, the number of inference streams <link> accordingly, so each device uses the ov::hint::num_requests when converting the hint to the actual device configuration options.
+For the explicit usage i.e. the BATCH device you can limit the batch size via "BATCH:GPU(4)",  when you have only 4 requests running in parallel.
+
+Limitations:
+ - Although less critical for the throughput-oriented scenarios, the load-time is increased almost 2x when auto-batching is used 
+ - Certain networks are not reshape-able by the batching (aka 'N', using the layouts parlance) dimension, so the auto-batching will not be triggered 
+ - Memory footpint growth is unavoidable price. Yet the auto-batching queries the available memory (especially for the dGPUs) and limits the selected batch size accordingly.
+
+ 
+
+### Configuring the Automatic Batching Plugin
+Following the OpenVINO convention for devices names, the  "batching" device is named as  "BATCH". The configuration options for that are as follows:
+
+| Parameter name     | Parameter description      | Default            |             Examples                                                      |
+| :---               | :---                  | :---               |:-----------------------------------------------------------------------------|
+| "AUTO_BATCH_DEVICE" | device name to apply the automatic batching and optional batch size in brackets | N/A | BATCH:GPU which triggers the automatic batch size selection or explicit batch size BATCH:GPU(4)     |
+| "AUTO_BATCH_TIMEOUT" | timeout value, in ms | 1000 |  you can reduce the timeout value (to avoid performance penalty when the data arrives too non-evenly) e.g. pass the "100", or in contrast make it large enough e.g. to accommodate inputs preparation (e.g. when it is serial process)     |
```

Comment:
```suggestion
| "AUTO_BATCH_TIMEOUT" | timeout value, ms | 1000 |  User can reduce the timeout value to avoid performance penalty when the data arrives non-evenly, for example, by passing *100*. Or, on the contrary, make it large enough to accommodate the preparation of inputs, when it happens as a serial process. |
```